docs/analyzers/Analyzer Samples.md

```diff
@@ -0,0 +1,30 @@
+﻿**Samples Location:**
+
+[This](..//..//src//Diagnostics//Samples) folder contains sample C# and VB analyzers to demonstrate recommended implementation models for different analysis scenarios.
```

Comment:
Doing 1 is sufficient I think. Having one sln is better than many and the inconsistenly is in a small detail.